### PR TITLE
UF-305 - Upgrade GwtBootstrap to version 0.9.3

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

**Comment by cristianonicolai** [comment]
Depends on https://github.com/droolsjbpm/guvnor/pull/344/files

File: kie-wb-common-screens/kie-wb-common-data-modeller/kie-wb-common-data-modeller-client/src/main/java/org/kie/workbench/common/screens/datamodeller/client/widgets/maindomain/MainDataObjectFieldEditorViewImpl.java [hunk 1]
```diff
@@ -79,19 +79,7 @@ public MainDataObjectFieldEditorViewImpl() {
...
-                //When the selected value changes, the ChangeEvent is raised two times by the Select component.
-                //It happens the same with all the Selects around the workbench.
-                //In this particular widget, since we have some post processing when selected value changes, etc., the
-                //duplicated event is causing issues.
-                //This event.stopPropagation() seems to stop the raising of the duplicated event.
-                event.stopPropagation();
-                presenter.onTypeChange();
-            }
-        } );
+        typeSelector.addValueChangeHandler( e -> presenter.onTypeChange() );
```
Comment (manstis) [inline kie-wb-common-screens/kie-wb-common-data-modeller/kie-wb-common-data-modeller-client/src/main/java/org/kie/workbench/common/screens/datamodeller/client/widgets/maindomain/MainDataObjectFieldEditorViewImpl.java]: Have you confirmed the workaround @wmedvede had above is no longer required?

**Comment by manstis** [comment]
+1 to merge (with others) but think @wmedvede should comment on removal of the workaround.

File: kie-wb-common-screens/kie-wb-common-data-modeller/kie-wb-common-data-modeller-client/src/main/java/org/kie/workbench/common/screens/datamodeller/client/widgets/maindomain/MainDataObjectFieldEditorViewImpl.java (hunk 1 above)
Comment (cristianonicolai) [inline kie-wb-common-screens/kie-wb-common-data-modeller/kie-wb-common-data-modeller-client/src/main/java/org/kie/workbench/common/screens/datamodeller/client/widgets/maindomain/MainDataObjectFieldEditorViewImpl.java]: @manstis sure, testing seems fine but I wouldn't mind a second opinion :)

**Pull request merged**